auth/src/Users.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+class Users extends React.Component {
+  render() {
+    return <div>Users</div>;
```

Comment:
here you would make your axios call to the users endpoint and set the response to state so you can map them and render them to the UI